readme.md

```diff
@@ -118,6 +118,51 @@ save your subscription.
       - **loginSuccess**  *array of functions*: function that can be plugged in to modify the behaviour of @digipolis/auth: function signature is the same as middleware `(req, res, next)`. these will run after successful login.
       - **logoutSuccess** *array of functions*: hooks that are triggered when logout is successful
 
+### Authentication 2.0
+If you want to use authentication 2.0 you can do so by adding `version: 'v2'` and add the necessary extra config.
+
+  - **auth2aprofiel** (optional if not needed):
+    - **version** *string*: authentication version you want to use. Defaults to v1.
+    - **minimalAssuranceLevel** *string*: Minimal Assurance Level. For now we only support `low` and `substantial`.
+    - **authMethods** *string*: the authentication methods you want to allow. (e.g. `iam-aprofiel-userpass` for simple username/password based authentication) 
```

Comment:
userpass will only work work with `minimalAssuranceLevel` `low` maybe make a note that you can not use auth methods that are not equal to or better than the requested `minimalAssuranceLevel`